src/main/java/picard/illumina/parser/readers/CbclReader.java

```diff
@@ -200,13 +200,12 @@ private void readSurfaceTile(final int tileNum, final List<AbstractIlluminaPosit
 
             for (final int outputLength : outputLengths) {
                 for (int cycle = 0; cycle < outputLength; cycle++) {
-                    final CycleData currentCycleData = cycleData[totalCycleCount];
```

Comment:
The code does use the expression `cycleData[totalCycleCount].tileInfo` three times below. If its value doesn't change in this scope, it would be clearer to store the result of that expression in a local variable.